spec/post_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(title: 'Test post 1', author_id: 1, comments_counter: 0, likes_counter: 0) }
+
+  before { subject.save }
+
+  it 'title should be present' do
+    subject.title = ''
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should not be too long' do
+    subject.title = 'a' * 270
+    expect(subject).to_not be_valid
+  end
+
+  it 'coments_counter should be integer' do
+    subject.comments_counter = '10'
+    expect(subject).to_not be_valid
+  end
+
+  it 'coments_counter should be greater than or equal to 0' do
+    subject.comments_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'likes_counter should be integer' do
+    subject.likes_counter = '10'
+    expect(subject).to_not be_valid
+  end
+
+  it 'likes_counter should be greater than or equal to 0' do
+    subject.likes_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'Return five most recent comments for post' do
+    expect(subject.recent_comments).to eq(subject.comments.order(created_at: :desc).limit(5))
+  end
+end
```

Comment:
- [ ] All unit tests should be passed. Here two of your tests showed failed. Kindly fix the issue. :+1: 
![rspec test](https://github.com/OsvaldoBC/blog/assets/70483835/4e735eed-625a-4e61-89f6-5bda8448e12d)